tests/custom_hypothesis_support.py

```diff
@@ -0,0 +1,42 @@
+import astroid
+import hypothesis
+import hypothesis.strategies as hs
+from typing import Any, Dict, List, Tuple
+
+# Custom strategies for hypothesis testing framework
+PRIMITIVE_TYPES = hs.sampled_from([
```

Comment:
Now that we are importing these, we should stick with the hypothesis style and not use all-caps. Instead, use the standard `snake_case` for naming these.